File: app/src/main/java/seveida/firetvforreddit/response/objects/Child.java
package seveida.firetvforreddit.response.objects;

import com.squareup.moshi.Json;

public class Child {

    @Json(name = "kind")
    public String kind;
    @Json(name = "data")
    public Data_ data;

    /**
     * No args constructor for use in serialization
     *
     */
    public Child() {
    }

    /**
     *
     * @param data
     * @param kind
     */
    public Child(String kind, Data_ data) {
        super();
        this.kind = kind;
        this.data = data;
    }

}

File: app/src/main/java/seveida/firetvforreddit/response/objects/Child.kt
package seveida.firetvforreddit.response.objects

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class Child(
    @Json(name = "kind") val kind: String,
    @Json(name = "data") val data: Data_
)
